pkg/grpc/template_config.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+
+option go_package = "github.com/DIMO-Network/vehicle-signal-decoding/pkg/grpc";
+
+package grpc;
+
+message Template {
+  string id  = 1;
+  string name = 2;
+  repeated string pids = 3;
+  string dbc = 4;
+  repeated string template_vehicles = 5;
+}
+message TemplateSummary {
+  string id = 1;
+  string name = 2;
+
```

Comment:
lets add the powertrain and protocol here too, how about also a hasDBC and PIDsCount? could be useful when we're looking at templates high level

Reply:
added